src/widget/OktaSignIn.js

```diff
@@ -142,6 +142,13 @@ var OktaSignIn = (function () {
 
     var authClient = options.authClient ? options.authClient : createAuthClient(authParams);
 
+    // validate authClient configuration against widget options
+    if (options.useInteractionCodeFlow && authClient.isPKCE() === false) {
+      throw new Errors.ConfigError(
+        'The "useInteractionCodeFlow" option requires PKCE to be enabled on the authClient.'
```

Comment:
Why don't we do this for the developer automatically?

Reply:
PKCE is enabled by default. If it is off, the developer has set it so explicitly. This is a config error, should only be seen by developers. Without this extra code, the error would be a runtime error, increasing the possibility that users would run into it because the developer thought it worked (but didn't try to login) then pushed it up.